gamestonk_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -38,13 +39,18 @@ class SectorIndustryAnalysisController(BaseController):
         "country",
         "mktcap",
         "exchange",
+        "period",
         "cps",
         "cpic",
         "cpis",
         "cpcs",
         "cpci",
         "sama",
         "metric",
+        "bs",
+        "cf",
+        "is",
+        "visualise",
     ]
```

Comment:
I would not have these 3 commands, they add a bit of noise. 

Instead, let's have a "satma       see all time metrics available" or similar, and then if the user does: `satma` they get a list of all the commands with some separation between balance sheet, cash flow or income statement. But then also allow the user to do `satma ic` to basically just see metrics associated with income statement.

Reply:
Fixed!